test/Air/Air.test.js

```diff
@@ -1740,6 +1740,26 @@ describe('#AirService', () => {
     });
   });
 
+  describe('providerReservationDivide', () => {
+    it('should check if correct function from service is called', () => {
+      const providerReservationDivide = sinon.spy(() => {});
+      const service = () => ({ providerReservationDivide });
+      const createAirService = proxyquire('../../src/Services/Air/Air', {
+        './AirService': service,
+      });
+      createAirService({ auth }).providerReservationDivide({
+        uapi_ur_locator: 'AAABBB',
+        version: '1',
+        provider: '1G',
+        pnr: '456ABC',
+        passengers: [
+          { uapi_ref_key: 'XX/ADFH2345AGJDS245IGJSDifgjoai22343s==' },
```

Comment:
It's `uapi_passenger_ref` in example and templates. How could this test work?

Reply:
renamed it to uapi_passenger_ref last minute, this must have been left out